Projects/Chat/Sources/Views/ChatScreenViewModel.swift

```diff
@@ -13,6 +13,7 @@ public class ChatScreenViewModel: ObservableObject {
     @Published var scrollToMessage: Message?
     @Published var banner: Markdown?
     @Published var conversationStatus: ConversationStatus = .open
+    @Published private var conversationId: String = ""
```

Comment:
Can we make this optinal? 
No need for published since we will not observe its value anywhere.